client/js/lounge.js

```diff
@@ -430,6 +430,14 @@ $(function() {
 		}
 	});
 
+	chat.on("click", ".condensed", function() {
+		$(this).toggleClass("closed");
+	});
+
+	chat.on("click", ".condensed div", function(e) {
```

Comment:
I didn't, this is what I get when I have such a long running PR and don't put everything in the to-do list. I'll hopefully have time tonight to try it.